### ISSUE TITLE
LatLonBox

### ISSUE THREAD

**Issue by sydying** [jumpinjackie/vscode-map-preview#47]
Hi, there  
I have an example from 
https://developers.google.com/kml/documentation/kml_tut 
as attached
[LatLonBox.txt](https://github.com/jumpinjackie/vscode-map-preview/files/4485735/LatLonBox.txt)

My guess is the map preview doesn't support LatLonBox tag?

Cheers, 
Ying

**Comment by jumpinjackie** [comment]
Yes, that is most likely explanation. The KML support in OpenLayers only covers geometry data within `<Placemark>` elements as far as I recall
